components/datadog/agent/kubernetes_helm.go

```diff
@@ -292,36 +292,36 @@ func buildLinuxHelmValues(installName, agentImagePath, agentImageTag, clusterAge
 				"agent": pulumi.Map{
 					"resources": pulumi.StringMapMap{
 						"requests": pulumi.StringMap{
-							"cpu":    pulumi.String("300m"),
+							"cpu":    pulumi.String("600m"),
```

Comment:
```suggestion
							"cpu":    pulumi.String("400m"),
```

@vboulineau 👍🏻 / 👎🏻 ?